lib/seadsa/DsaPrinter.cc

```diff
@@ -347,7 +347,7 @@ struct DOTGraphTraits<seadsa::Graph *> : public DefaultDOTGraphTraits {
     for (const llvm::Value *AS : Node.getAllocSites()) {
       assert(AS);
       ++i;
-      O << " " << DOT::EscapeString(AS->hasName() ? AS->getName() : "unnamed");
+      O << " " << DOT::EscapeString(AS->hasName() ? AS->getName().str() : "unnamed");
```

Comment:
seems strange to create `.str()` just to print. `getName()` should be printable.

Reply:
It seems `llvm::StringRef` doesn't overload the `<<` operator.

Reply:
that would be strange. more likely missing include. The operator would be defined outside of StringRef class.

Reply:
My fault. Actually, we're talking about the argument of `EscapeString` which should be a string.